cpp/deplex/src/deplex/cell_segment.h

```diff
@@ -16,10 +16,13 @@
 #pragma once
 
 #include <cstdint>
+#include <thread>
 
 #include "cell_segment_stat.h"
 #include "deplex/config.h"
 
+#define sizeThreads 4 // Optimal 4 - 8
```

Comment:
Why do we need to define this variable?

Reply:
This variable is necessary for calculating steps in parallel algorithms

Reply:
What I mean is why we need to use "define" directive. Can't we configure it dynamically or move to config file or configure it with CMake?

Reply:
Moved